### PR TITLE
Upgrade argocd -> v2.5.6

### PR DESCRIPTION
**Pull request by eduartua**

### CODE REVIEW

File: argocd-git-crypt/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM argoproj/argocd:v2.3.4
+FROM argoproj/argocd:v2.5.6
 
 # Switch to root for the ability to perform install
 USER root
 
-ARG GIT_CRYPT_VERSION=0.6.0-1build2
+ARG GIT_CRYPT_VERSION=0.6.0-1ubuntu1
```
Comment (victortrac) [inline argocd-git-crypt/Dockerfile:6]: What's the reason behind this change?

Comment (eduartua) [reply argocd-git-crypt/Dockerfile:6]: The git-crypt version `0.6.0-1build2` doesn't exist on the argocd image base ([jammy (22.04LTS)](https://packages.ubuntu.com/jammy/git-crypt)) package repositories. Therefore, docker build fails. I think the maintainer changed the package name. 
image
If you try to build with  `ARG GIT_CRYPT_VERSION=0.6.0-1build2` it'll fail.

Comment (eduartua) [reply argocd-git-crypt/Dockerfile:6]: @victortrac ping

**Review by victortrac — APPROVED** [review]
